web-client/src/components/CurrentVersionWithReplyToReview.jsx

```diff
@@ -0,0 +1,254 @@
+import Rating from '@material-ui/lab/Rating';
+import { TextField } from '@material-ui/core';
+import * as yup from "yup";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import InputAdornment from "@material-ui/core/InputAdornment";
+
+function CurrentVersionWithReplyToReview(props)
+{
+    const styles = makeStyles({
+        currentVersion:
+        {
+            width: '70%',
+            float: 'left',
+            marginTop: '5%',
+            textAlign: 'left',
+            marginLeft: '12%',
+        },
+        review:
+        {
+            width: '100%',
+            float: 'left',
+        },
+        leftReview:
+        {
+            width: '15%',
+            float: 'left',
+        },
+        rightReview:
+        {
+            width: '80%',
+            float: 'right',
+        },
+        image:
+        {
+            width: 80,
+            height: 80,
+            borderRadius: '50%',
+            float: 'center',
+            marginLeft: '12%',
+        },
+        me:
+        {
+            width: '100%',
+            float: 'left',
+            textAlign: 'center',
+        },
+        rightDate:
+        {
+            float: 'right',
+            color: 'grey',
+        },
+        textReview:
+        {
+            width: '95%',
+            float: 'left',
+            paddingLeft: '2%',
+            fontSize: '16px',
+            paddingBottom: '2%',
+        },
+        reply:
+        {
+            width: '80%',
+            float: 'left',
+            marginLeft: '20%',
+            marginTop: '5%',
+        },
+        replyToReview:
+        {
+            width: '100%',
+            fontWeight: 'bold',
+            float: 'left',
+            fontSize: '20px',
+            marginLeft: '2%',
+        },
+        leftReply:
+        {
+            width: '18%',
+            float: 'left',
+            marginTop: '2.5%',
+        },
+        rightReply:
+        {
+            width: '80%',
+            float: 'right',
+        },
+        textField:
+        {
+            width: '95%',
+            margin: '2%',
+        },
+        info:
+        {
+            marginLeft: '5%',
+            fontSize: '14px',
+            float: 'left',
+            color:'grey',
+        },
+        btn:
+        {
+            textTransform: 'none',
+        },
+        btn2:
+        {
+            textTransform: 'none',
+            marginRight: '2.5%',
+            float: 'right',
+        },
+        yourReplay:
+        {
+            width: '50%',
+            fontWeight: 'bold',
+            float: 'left',
+            fontSize: '20px',
+            marginLeft: '2%',
+        },
+        message:
+        {
+            width: '100%',
+            float: 'left',
+            marginLeft: '2%',
+            marginTop: '3%',
+        },
+    });
+
+    const style = styles();
+    
+    const [value] = useState(props.stars);  
+    const [hover] = useState(3);
```

Comment:
How about this?
```suggestion
    const value = props.stars;  
    const hover = 3;
```

But still I'm little confused with these variables :/ Especially with `values`, do we need it?

Reply:
yes you are right i even changed to const hover = props.stars;

I explained before.